src/benchmarklib/abstract_table_generator.cpp

```diff
@@ -448,4 +446,48 @@ std::unordered_map<std::string, BenchmarkTableInfo> AbstractTableGenerator::_loa
   return table_info_by_name;
 }
 
+std::unordered_map<std::string, BenchmarkTableInfo> AbstractTableGenerator::_load_binary_tables_from_json() {
+  std::unordered_map<std::string, BenchmarkTableInfo> table_info_by_name;
+  auto& storage_manager = Hyrise::get().storage_manager;
+  const auto tables_files_mapping = storage_manager.get_tables_files_mapping();
```

Comment:
This should be included in the time calculation.

Reply:
What does this mean? In which time calculation?